armi/reactor/blueprints/componentBlueprint.py

```diff
@@ -248,38 +248,44 @@ def _setComponentCustomDensity(self, comp, blueprint, matMods, inputHeightsConsi
 
         if densityFromCustomIsotopic <= 0:
             runLog.error(
-                "A zero or negative density was specified in a custom isotopics input. "
-                "This is not permitted, if a 0 density material is needed, use 'Void'. "
-                f"The component is {comp} and the isotopics entry is {self.isotopics}."
+                "A zero or negative density was specified in a custom isotopics input. This is not permitted, if a 0 "
+                f"density material is needed, use 'Void'. The component is {comp} and the isotopics entry is "
+                f"{self.isotopics}."
             )
             raise ValueError("A zero or negative density was specified in the custom isotopics for a component")
+        elif len(matMods):
+            runLog.warning(
+                "Custom isotopics and material modifications have both been defined for {self.material}. Please "
```

Comment:
```suggestion
                f"Custom isotopics and material modifications have both been defined for {self.material}. Please "
```

Reply:
Also it might be better to tell the user which component this is, not just the material.

Reply:
done